pallets/did/src/default_weights.rs

```diff
@@ -63,167 +63,187 @@ pub trait WeightInfo {
-		(20_557_000_u64)
-			// Standard Error: 11_000
-			.saturating_add((2_816_000_u64).saturating_mul(n as Weight))
-			// Standard Error: 11_000
-			.saturating_add((1_874_000_u64).saturating_mul(m as Weight))
+	fn update_ecdsa_keys(n: u32, m: u32, u: u32, c: u32, ) -> Weight {
+		(22_988_000_u64)
+			// Standard Error: 14_000
+			.saturating_add((2_823_000_u64).saturating_mul(n as Weight))
+			// Standard Error: 14_000
+			.saturating_add((1_793_000_u64).saturating_mul(m as Weight))
 			// Standard Error: 0
-			.saturating_add((3_000_u64).saturating_mul(u as Weight))
+			.saturating_add((4_000_u64).saturating_mul(u as Weight))
+			// Standard Error: 64_000
+			.saturating_add((319_000_u64).saturating_mul(c as Weight))
 			.saturating_add(T::DbWeight::get().reads(1_u64))
 			.saturating_add(T::DbWeight::get().writes(1_u64))
 	}
 	fn delete() -> Weight {
-		(17_563_000_u64)
+		(17_904_000_u64)
 			.saturating_add(T::DbWeight::get().reads(1_u64))
 			.saturating_add(T::DbWeight::get().writes(1_u64))
 	}
 	fn submit_did_call_ed25519_key() -> Weight {
-		(66_766_000_u64)
+		(67_156_000_u64)
 			.saturating_add(T::DbWeight::get().reads(1_u64))
 			.saturating_add(T::DbWeight::get().writes(1_u64))
 	}
 	fn submit_did_call_sr25519_key() -> Weight {
-		(68_679_000_u64)
+		(70_993_000_u64)
 			.saturating_add(T::DbWeight::get().reads(1_u64))
 			.saturating_add(T::DbWeight::get().writes(1_u64))
 	}
 	fn submit_did_call_ecdsa_key() -> Weight {
-		(170_040_000_u64)
+		(170_751_000_u64)
 			.saturating_add(T::DbWeight::get().reads(1_u64))
 			.saturating_add(T::DbWeight::get().writes(1_u64))
 	}
 }
 
 // For backwards compatibility and tests
 impl WeightInfo for () {
-	fn create_ed25519_keys(n: u32, u: u32, ) -> Weight {
-		(77_652_000_u64)
-			// Standard Error: 35_000
-			.saturating_add((1_627_000_u64).saturating_mul(n as Weight))
-			// Standard Error: 1_000
-			.saturating_add((7_000_u64).saturating_mul(u as Weight))
+	fn create_ed25519_keys(n: u32, u: u32, c: u32, ) -> Weight {
+		(71_462_000_u64)
+			// Standard Error: 105_000
+			.saturating_add((2_176_000_u64).saturating_mul(n as Weight))
+			// Standard Error: 4_000
+			.saturating_add((19_000_u64).saturating_mul(u as Weight))
+			// Standard Error: 475_000
+			.saturating_add((2_897_000_u64).saturating_mul(c as Weight))
 			.saturating_add(RocksDbWeight::get().reads(1_u64))
 			.saturating_add(RocksDbWeight::get().writes(1_u64))
 	}
-	fn create_sr25519_keys(n: u32, u: u32, ) -> Weight {
-		(77_617_000_u64)
-			// Standard Error: 43_000
-			.saturating_add((1_950_000_u64).saturating_mul(n as Weight))
-			// Standard Error: 2_000
-			.saturating_add((2_000_u64).saturating_mul(u as Weight))
+	fn create_sr25519_keys(n: u32, u: u32, c: u32, ) -> Weight {
+		(80_034_000_u64)
+			// Standard Error: 111_000
+			.saturating_add((1_580_000_u64).saturating_mul(n as Weight))
+			// Standard Error: 5_000
+			.saturating_add((20_000_u64).saturating_mul(u as Weight))
+			// Standard Error: 502_000
```

Comment:
the standard error looks quite high here? Are there any reasons for that?

Reply:
For the peregrine runtime it is overall lower, so I guess it's because we are running two local parachain setups on the Rust server?

Reply:
but not on our bench server?

Reply:
Ah ok you run those on the rust server!? Ok that would explain it

Reply:
I run the default ones on the Rust server, but the runtime ones on the bench server. Should I run also the default ones on the bench server?

Reply:
I guess I could just copy-paste the result then, no?

Reply:
just leave it. The default once don't matter and are unused anyway. 🤷🏻